vllm/model_executor/models/minicpmv.py

```diff
@@ -1007,20 +1003,11 @@ def __new__(
         cache_config: Optional[CacheConfig] = None,
         quant_config: Optional[QuantizationConfig] = None,
     ):
-        if not hasattr(config, "version"):
-            if config.hidden_size == 2304 and config.query_num == 64:
-                version = (2, 0)
-            else:
-                version = (2, 5)
-        else:
-            version = str(config.version).split(".")
-            version = tuple([int(x) for x in version])
+        version = str(config.version).split(".")
```

Comment:
Has the config for v2.0 been updated yet?

Reply:
Not yet, my mistake